test_scripts/API/PerformAudioPassThru/013_ATF_N_PerformAudioPassThru_INVALID_DATA_All_Params_audioPassThruIcon_STATIC.lua

```diff
@@ -0,0 +1,121 @@
+-- [GeneralResultCodes] INVALID_DATA wrong characters
+-- [GeneralResultCodes] INVALID_DATA empty String parameter 
+-- [GeneralResultCodes] INVALID_DATA wrong type 
+-- [GeneralResultCodes] INVALID_DATA out of bounds 
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case that the request comes with 
+-- '\n' and-or '\t' and-or 'whitespace'-as-the-only-symbol(s) at any "String" type parameter in the request structure, 
+-- or with empty value"" in "String" type parameters (including parameters of the structures)
+-- or wrong type parameters (including parameters of the structures)
+-- or out-of-bounds parameters values of any type 
+-- SDL must respond with resultCode "INVALID_DATA" and success:"false"
+--
+-- 1. Used preconditions
+-- 1.1. Request is sent with audioPassThruIcon image structure with invalid value parameter
+-- and ImageType = STATIC, all other params used in PerformAudioPassThru_request 
+-- are present and within bounds 
+--
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon (invalid value parameter,ImageType = STATIC), other params) from mobile to SDL and check:
+--
+-- Expected result:
+-- SDL sends PerformAudioPassThru (INVALID_DATA, success:false) to mobile app
+
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPostconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ Local variables ]]
+local params_invalid_data = 
+{
+  {param_value = "   ", comment = "audioPassThruIcon_only_whitespaces"}, -- audioPassThruIcon contains only whitespaces in value parameter 
+  {param_value = "iconvalue\newlinechar", comment = "audioPassThruIcon_new_line_char"}, -- audioPassThruIcon contains new line character in value parameter
+  {param_value = "iconvalue\ttabchar", comment = "audioPassThruIcon_tab_char"}, -- audioPassThruIcon contains tab character in value parameter
+  {param_value = "", comment = "audioPassThruIcon_empty"}, -- audioPassThruIcon with empty value parameter
+  {param_value = "123", comment = "audioPassThruIcon_wrong_type"}, --audioPassThruIcon contains wrong type (integer instead of string) in value parameter
```

Comment:
@maria-mih, in this way declaration type is again string and this is not wrong type.
You should declare as: param_value = 123